geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/jts/util/util.scala

```diff
@@ -0,0 +1,10 @@
+package org.locationtech.geomesa.spark.jts.util
+
+import com.vividsolutions.jts.geom.{Geometry, LineString, Point, Polygon}
+
+package object util {
+  case class PointContainer(geom: Point)
```

Comment:
I'm slightly confused (probably due to my unfamiliarity with spark), but why are these containers needed?

Reply:
The original cast error surfaced when DataFrames (`type DataFrame = Dataset[Row]`) were converted to `Dataset[Container]` (using `df.as[Container]`) in order to use rows as strongly typed objects.

More here:
https://databricks.com/blog/2016/07/14/a-tale-of-three-apache-spark-apis-rdds-dataframes-and-datasets.html

Reply:
Cool.  Am I understanding this correctly that this adds better DataSet support to the JTS+Spark work here?